vendorconsent/tcf2/pubrestrict_test.go

```diff
@@ -0,0 +1,38 @@
+package vendorconsent
+
+import (
+	"testing"
+)
+
+func TestPubRestrictions(t *testing.T) {
+	baseConsent, err := Parse(decode(t, "COwAdDhOwAdDhN4ABAENAPCgAAQAAv___wAAAFP_AAp_4AI6ACACAA"))
```

Comment:
Please break into two tests (one per Parse method) and add descriptions to explain the asserts.

Reply:
Descriptions in the errors would require refactoring all `assertBoolsEqual` calls. Or do you mean comments in the code? I thought that was clear enough.

Reply:
Yeah, not in the assert statements. To clarify, I'm asking for comment descriptions to explain the difference of what's between tested between the two test strings.